docs/core/porting/third-party-deps.md

```diff
@@ -63,30 +68,59 @@ dotnet5.4
-2. You can contact the author directly on [nuget.org](https://www.nuget.org/) by searching for the package and clicking "Contact Owners" on the left hand side of the package's page.
-3. You can look for another package that runs on .NET Core which accomplishes the same task as the package you were using.
+1. If the project is open source and hosted somewhere like GitHub, you can engage the developers directly.
+2. You can contact the author directly on [nuget.org](https://www.nuget.org/). Search for the package and click **Contact Owners** on the left-hand side of the package's page.
+3. You can search for another package that runs on .NET Core that accomplishes the same task as the package you were using.
 4. You can attempt to write the code the package was doing yourself.
-5. You could eliminate the dependency on the package by changing the functionality of your app, at least until a compatible version of the package becomes available.
+5. You could eliminate the dependency on the package by changing the functionality of your app. At least, until a compatible version of the package becomes available.
 
-Please remember that open source project maintainers and NuGet package publishers are often volunteers who contribute because they care about a given domain, do it for free, and often have a different daytime job. If you do reach out, you might start with a positive statement about the library before asking about .NET Core support.
+Remember that open-source project maintainers and NuGet package publishers are often volunteers. They contribute because they care about a given domain, do it for free, and often have a different daytime job. So, be mindful of that when contacting them to ask for .NET Core support.
 
-If you're unable to resolve your issue with any of the above, you may have to port to .NET Core at a later date.
+If you can't resolve your issue with any of the above, you may have to port to .NET Core at a later date.
 
-The .NET Team would like to know which libraries are the most important to support next with .NET Core. You can also send us mail at dotnet@microsoft.com about the libraries you'd like to use.
+The .NET Team would like to know which libraries are the most important to support next with .NET Core. You can send an email to dotnet@microsoft.com about the libraries you'd like to use.
 
-## Analyzing Dependencies which aren't NuGet Packages
+## Analyze dependencies that aren't NuGet packages
 
-You may have a dependency that isn't a NuGet package, such as a DLL in the filesystem.  The only way to determine the portability of that dependency is to run the [ApiPort tool](https://github.com/Microsoft/dotnet-apiport/blob/master/docs/HowTo/).
+You may have a dependency that isn't a NuGet package, such as a DLL in the filesystem. The only way to determine the portability of that dependency is to run the [ApiPort tool](https://github.com/Microsoft/dotnet-apiport/blob/master/docs/HowTo/).
```

Comment:
filesystem --> filesystem
Add a parenthetical description of what the APIPort tool does.

Reply:
I think you mean to change it to file system?